index.js

```diff
@@ -19,6 +22,29 @@ function underPressure (fastify, opts, next) {
   var eventLoopDelay = 0
   var lastCheck = now()
   const timer = setInterval(updateMemoryUsage, sampleInterval)
+  timer.unref()
+
+  var externalsHealthy = false
+  var externalHealthCheckTimer
+  if (healthCheck) {
+    assert(typeof healthCheck === 'function', 'opts.healthCheck should be a function that returns a promise that resolves to true or false')
+
+    const doCheck = async () => {
+      try {
+        externalsHealthy = await healthCheck()
+      } catch (error) {
+        externalsHealthy = false
+        fastify.log.error('external healthCheck function supplied to `under-pressure` threw an error. setting the service status to unhealthy.', { error })
```

Comment:
You should put { err } as the beginning of the call to .error to use the error serializer